src/custom-code-helpers/domain-lock/templates/DomainLockTemplate.ts

```diff
@@ -7,59 +7,56 @@ export function DomainLockTemplate (): string {
             
             {globalVariableTemplate}
             
-            const func = function () {
-                return {
-                    key: 'item',
-                    value: 'attribute',
-                    getAttribute: function () {
-                        for (let i = 0; i < 1000; i--) {
-                            const isPositive = i > 0;
-                            
-                            switch (isPositive) {
-                                case true:
-                                    return this.item + '_' + this.value + '_' + i;
-                                default:
-                                    this.item + '_' + this.value;
-                            }
-                        }
-                    }()
-                };
-            };
-                        
-            const regExp = new RegExp("[{diff}]", "g");
+            const regExp = new RegExp("[{domainsStringDiff}]", "g");
             const domains = "{domains}".replace(regExp, "").split(";");
             let document;
             let domain;
             let location;
             let hostname;
 
+            const isName = function(name, length, cs) {
```

Comment:
I think that of course, moving repeated code to function is good practce in usual coding, but in obfuscation template more unclear is better.

Reply:
I thought the repeating code made it easier to find this in the obfuscated code?

Reply:
Okay, my opinion changed, now I agree with you😃

Reply:
How about make proxy function of `isName`?

```
const isName = function(name, length, cs) {
  // ...
};

const isName_2 = function(length, name, cs) {
  return isName(name, length, cs);
};
```

Reply:
What would the benefit of `isName_2` be?

Reply:
> What would the benefit of `isName_2` be?

More obscure, different variable names and arguments order. Similar is used in StringArrayHelper.

Reply:
I have fixed this and updated this branch with master. What else needs to change for this to be merged?

Reply:
Added some comments. Please write some documentation about how this option works in the `README.md` options section. I will add more things to the `README.md` after the PR will be merged, but the main description is desirable.